cli/src/main.rs

```diff
@@ -398,7 +405,7 @@ fn run() -> Result<(), CliError> {
                             .value_name("circuit-id")
                             .takes_value(true)
                             .required(true)
-                            .help("The circuit id of the proposed circuit"),
+                            .help("Circuit id of the proposed circuit"),
```

Comment:
Could even just be "ID of the proposed circuit" (same below)

Reply:
Addressed